### PR TITLE
CONSOLE-3428: blocked-edges: Declare ConsoleAvailableUpdatesNull for 4.12.1 and 4.13.0-ec.2

### PR DESCRIPTION
**Pull request by wking**

### CODE REVIEW

File: blocked-edges/4.12.1-ConsoleAvailableUpdatesNull.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+to: 4.12.1
+from: .*
+url: https://issues.redhat.com/browse/CONSOLE-3428
+name: ConsoleAvailableUpdatesNull
+message: |-
+  An administrator console regression breaks the cluster settings page when 'availableUpdates' is null.  You can still use 'oc adm upgrade', or wait for a fixed relaase.
```
Comment (petr-muller) [inline blocked-edges/4.12.1-ConsoleAvailableUpdatesNull.yaml:6]:
```suggestion
  An administrator console regression breaks the cluster settings page when 'availableUpdates' is null.  You can still use 'oc adm upgrade', or wait for a fixed release.
```

File: blocked-edges/4.13.0-ec.2-ConsoleAvailableUpdatesNull.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+to: 4.13.0-ec.2
+from: .*
+url: https://issues.redhat.com/browse/CONSOLE-3428
+name: ConsoleAvailableUpdatesNull
+message: |-
+  An administrator console regression breaks the cluster settings page when 'availableUpdates' is null.  You can still use 'oc adm upgrade', or wait for a fixed relaase.
```
Comment (petr-muller) [inline blocked-edges/4.13.0-ec.2-ConsoleAvailableUpdatesNull.yaml:6]:
```suggestion
  An administrator console regression breaks the cluster settings page when 'availableUpdates' is null.  You can still use 'oc adm upgrade', or wait for a fixed release.
```

File: blocked-edges/4.12.1-ConsoleAvailableUpdatesNull.yaml (hunk 1 above)
Comment (wking) [reply blocked-edges/4.12.1-ConsoleAvailableUpdatesNull.yaml]: Nice catch; fixed with 6bc47705 -> 5bb1481f, which also covers some `Upgradeable=False` tie-ins.

**Review by petr-muller — APPROVED** [review]
/hold

LGTM, merge as needed

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**